lib/vast/Dialect/HighLevel/Passes.cpp

```diff
@@ -52,7 +53,7 @@ namespace vast::hl::pipeline {
     }
 
     pipeline_step_ptr simplify() {
-        return compose("simplify", ude, dce, desugar);
+        return compose("simplify", conv::pipeline::to_hlbi, ude, dce, desugar);
```

Comment:
call this after ude and dce as it will have less work

Reply:
after to_hlbi more function declarations become unused, that's why I put it before ude